filebeat/module/santa/_meta/kibana/7/dashboard/filebeat-santa-log-overview.json

```diff
@@ -1 +1 @@
-11e8-93c5-d5ecd1b3e307-ecs","type":"visualization","updated_at":"2018-12-14T06:57:58.885Z","version":2},{"attributes":{"columns":["agent.hostname","process.executable","user.name","certificate.common_name"],"description":"","hits":0,"kibanaSavedObjectMeta":{"searchSourceJSON":{"filter":[{"$state":{"store":"appState"},"meta":{"alias":null,"disabled":false,"index":"filebeat-*","key":"event.module","negate":false,"params":{"query":"santa","type":"phrase"},"type":"phrase","value":"santa"},"query":{"match":{"event.module":{"query":"santa","type":"phrase"}}}},{"$state":{"store":"appState"},"meta":{"alias":null,"disabled":false,"index":"filebeat-*","key":"event.dataset","negate":false,"params":{"query":"log","type":"phrase"},"type":"phrase","value":"log"},"query":{"match":{"event.dataset":{"query":"log","type":"phrase"}}}}],"highlightAll":true,"index":"filebeat-*","query":{"language":"kuery","query":""},"version":true}},"sort":["@timestamp","desc"],"title":"Santa Logs Search [Filebeat Santa] ECS","version":1},"id":"6d56a010-ff6a-11e8-93c5-d5ecd1b3e307-ecs","type":"search","updated_at":"2018-12-14T06:57:11.037Z","version":2},{"attributes":{"description":"Process executions on macOS monitored by Google Santa.","hits":0,"kibanaSavedObjectMeta":{"searchSourceJSON":{"filter":[],"query":{"language":"kuery","query":""}}},"optionsJSON":{"darkTheme":false,"hidePanelTitles":false,"useMargins":true},"panelsJSON":[{"embeddableConfig":{},"gridData":{"h":12,"i":"1","w":10,"x":0,"y":0},"id":"dad521d0-ff69-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"1","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":12,"i":"2","w":38,"x":10,"y":0},"id":"1579d690-ff6b-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"2","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":10,"i":"3","w":10,"x":8,"y":12},"id":"51677b80-ff6b-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"3","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":10,"i":"4","w":12,"x":36,"y":12},"id":"30962fe0-ff6c-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"4","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":10,"i":"5","w":8,"x":0,"y":12},"id":"b06c0460-ff6c-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"5","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":10,"i":"6","w":18,"x":18,"y":12},"id":"11858000-ff6d-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"6","type":"visualization","version":"7.0.0-alpha1-SNAPSHOT"},{"embeddableConfig":{},"gridData":{"h":10,"i":"7","w":48,"x":0,"y":22},"id":"6d56a010-ff6a-11e8-93c5-d5ecd1b3e307-ecs","panelIndex":"7","type":"search","version":"7.0.0-alpha1-SNAPSHOT"}],"timeRestore":false,"title":"[Filebeat Santa] Overview ECS","version":1},"id":"161855f0-ff6a-11e8-93c5-d5ecd1b3e307-ecs","type":"dashboard","updated_at":"2018-12-14T06:58:23.367Z","version":5}],"version":"7.0.0-alpha1-SNAPSHOT"}
```

Comment:
@andrewkroh Just stumbled over his JSOn here which is all on one line. Is this on purpose?

Reply:
Resolved in an other PR so we can see the diff.